ALVRTrackingInterface/ALVRTrackingInterface.cs

```diff
@@ -270,25 +272,22 @@ private void UpdateEyeData(ref UnifiedEyeData eye, ref float[] expressions)
                 LookRight: expressions[(int)FBExpression.Eyes_Look_Right_R],
                 LookUp: expressions[(int)FBExpression.Eyes_Look_Up_R],
                 LookDown: expressions[(int)FBExpression.Eyes_Look_Down_R],
-                Openness: Math.Min(1, eye.Right.Openness)
+                Openness: Math.Min(1, OpennessRight)
             );
 
             // Eye dilation code, automated process maybe?
-            eye.Left.PupilDiameter_MM  = 0.0035f;
+            eye.Left.PupilDiameter_MM = 0.0035f;
             eye.Right.PupilDiameter_MM = 0.0035f;
 
             #endregion
-        }
 
-        private void UpdateEyeExpressions(ref UnifiedExpressionShape[] unifiedExpressions, ref float[] expressions)
-        {
             #region Eye Expressions Set
 
-            unifiedExpressions[(int)UnifiedExpressions.EyeWideLeft].Weight = expressions[(int)FBExpression.Upper_Lid_Raiser_L] ;
-            unifiedExpressions[(int)UnifiedExpressions.EyeWideRight].Weight = expressions[(int)FBExpression.Upper_Lid_Raiser_R] ;
+            unifiedExpressions[(int)UnifiedExpressions.EyeWideLeft].Weight = (float)Math.Pow(1.0f - expressions[(int)FBExpression.Eyes_Closed_L], 1.0f / EyeLidWeight) * (float)Math.Pow(expressions[(int)FBExpression.Upper_Lid_Raiser_L], 1.0 / WidenWeight);
```

Comment:
Same as mention as above, Widen does not need any compensation.

Reply:
Basically for this module the *only* parameter that needs to be compensated for is Openness, and AFAIK only lid tightener and eye close should be the only ones being weighted into Openness. Every other parameter should be 1:1. We shouldn't expect every module to compensate for v1 parameters at all.

Reply:
If you're okay with that I am going to make some changes to revert some of the code back, and feed Openness like what I am doing with the MeowFace module. I will formulate the ExpandedSqueeze to work best with that approach.

Reply:
That will work